### ISSUE TITLE
"+ x more" appointsments can overflow in RadzenScheduler

### ISSUE THREAD

**Issue by OleFosdal** [radzenhq/radzen-blazor#1859]
**Describe the bug**
"+ x more" appointsments can overflow when RadzenScheduler is set to heights different from example on demos site.
This will also result in RadzenScheduler showing ScrollBar if bottom row of cells shows the "+ x more" UI.

**To Reproduce**
1. Go to [radzen scheduler](https://blazor.radzen.com/scheduler?theme=material3)
2. Modify RadzenScheduler height to "800px"
3. Make enough appointments in any cell so that the text "+x more" appointments shows up.

**Expected behavior**

**Screenshots**
![image](https://github.com/user-attachments/assets/886e6762-b5d1-4dcf-bd4d-179b05597fa7)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser chrome 131.0.6778.140

**Comment by akorchev** [comment]
Yep, this happens because the calculation that does MaxAppointmentsPerSlot is a bit iffy. We will see if there is any way to make it more robust somehow. As a workaround you can set MaxAppointmentsPerSlot (it is a property of RadzenMonthView). I think 3 should be fine.
